src/Pages/2DPrinting/2DPrinting.js

```diff
@@ -49,15 +54,87 @@ export default function Printing(props) {
     props.user.pagesPrinted
   );
   const [previewDisplay, setPreviewDisplay] = useState('');
+  const [loadPreview, setLoadPreview] = useState(true);
+
   const previewLabels = {
     copies: 'Number of Copies',
     sides: 'Type of print',
     pages: 'Pages'
   };
+
   const headerProps = {
     title: 'Printing'
   };
 
+  const continueButtonProps = {
+    color: 'primary',
+    className: 'continue',
+    hidden: !continueButn,
+    onClick: () => {
+      handleCanPrint(usedPages, copies);
+      setPreviewModal(true);
+    },
+    text: 'Continue'
+  };
+
+  const backButtonProps = {
+    color: 'danger',
+    onClick: () => {
+      setPreviewModal(false);
+      setPages(false);
+      let arr = new Set(range(1, numPages + 1));
+      setUsedPages(arr);
+    },
+    text: 'Back'
+  };
+
+  const printButtonProps = {
+    color: 'success',
+    onClick: () => {
+      setConfirmModal(!confirmModal);
+    },
+    disabled: !canPrint,
+    text: 'Print!'
+  };
+
+  const previewModalProps = {
+    isOpen: previewModal,
```

Comment:
i dont think this is a valid prop, see chrome output
![image](https://user-images.githubusercontent.com/36345325/85951252-d4de9f00-b916-11ea-91b1-25fffa3f1347.png)